src/Sarif/IAnalysisLogger.cs

```diff
@@ -11,6 +11,10 @@ public interface IAnalysisLogger
 
         void AnalyzingTarget(IAnalysisContext context);
 
+        void TargetAnalysisCompleted(IAnalysisContext context);
+
+        void LogMemoryUsage(IAnalysisContext context);
```

Comment:
>LogMemoryUsage [](http://example.com/codeflow?start=13&length=14)

Why is this interface change still here? You should eliminate it and instead generate a tool notification using the logger at the appropriate place.